### PR TITLE
Fix typo with the `TimerKey`

### PR DESCRIPTION
**Pull request by eaba**
## Changes

Fix typo with the `TimerKey` which was `GETRECORDS` instead of `GET_RECORDS` as a result after the first batch of messages are consumed, the timer fails to fetch the next batch of messages because, `RequestRecords(_self)` is never called for the second time!

### CODE REVIEW

File: src/AWS/Kinesis/Akka.Streams.Kinesis/KinesisSourceStage.cs [hunk 1]
```diff
@@ -202,7 +202,7 @@ private void Ready((IActorRef actorRef, object message) args)
 
             protected override void OnTimer(object timerKey)
             {
-                if ("GETRECORDS" == (string)timerKey)
+                if ("GET_RECORDS" == (string)timerKey)
```
Comment (Aaronontheweb) [inline src/AWS/Kinesis/Akka.Streams.Kinesis/KinesisSourceStage.cs]: Can we stick this into a `const` that is shared between both places?

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]

**Comment by Aaronontheweb** [comment]
Also, need to harden the test suite to ensure that the Source has to consume multiple batches prior to completion.

File: src/AWS/Kinesis/Akka.Streams.Kinesis/KinesisSourceStage.cs (hunk 1 above)
Comment (eaba) [reply src/AWS/Kinesis/Akka.Streams.Kinesis/KinesisSourceStage.cs]: Sure!

File: src/AWS/Kinesis/Akka.Streams.Kinesis.Tests/KinesisSourceSpec.cs [hunk 2]
```diff
@@ -110,7 +110,48 @@ public void KinesisSource_must_poll_for_records_with_multiple_requests()
             probe.ExpectNext("a");
             probe.ExpectNext("b");
         }
+        [Fact]
+        public void KinesisSource_must_poll_for_batched_records_with_multiple_requests()
```
Comment (eaba) [inline src/AWS/Kinesis/Akka.Streams.Kinesis.Tests/KinesisSourceSpec.cs:114]: Wasn't able to test this locally, so testing with the build server!

**Pull request merged**